src/prettytable/prettytable.py

```diff
@@ -513,9 +519,9 @@ def _validate_align(self, val):
 
     def _validate_valign(self, val):
         try:
-            assert val in ["t", "m", "b", None]
+            assert val in ["t", "m", "b"]
         except AssertionError:
-            msg = f"Alignment {val} is invalid, use t, m, b or None"
+            msg = f"Alignment {val} is invalid, use t, m, b"
             raise ValueError(msg)
```

Comment:
It's still possible to set `valign` to `None`. That's just handled by the property handler. The validation method will never get `None`.
https://github.com/jazzband/prettytable/blob/c7c89cf336755b3ed6662e4ad322bf980d9a06a6/src/prettytable/prettytable.py#L719-L729